apps/outlook-rooms-addin/manifest.xml

```diff
@@ -79,21 +89,26 @@
     </Hosts>
     <Resources>
       <bt:Images>
-        <bt:Image id="Icon.16x16" DefaultValue="https://127.0.0.1:3000/assets/icon-16.png"/>
-        <bt:Image id="Icon.32x32" DefaultValue="https://127.0.0.1:3000/assets/icon-32.png"/>
-        <bt:Image id="Icon.80x80" DefaultValue="https://127.0.0.1:3000/assets/icon-80.png"/>
+        <bt:Image id="Icon.16x16" DefaultValue="https://s3.ap-southeast-2.amazonaws.com/os.place.tech/outlook-plugin-resources/16x16-01.png"/>
+        <bt:Image id="Icon.32x32" DefaultValue="https://s3.ap-southeast-2.amazonaws.com/os.place.tech/outlook-plugin-resources/32x32-01.png"/>
+        <bt:Image id="Icon.80x80" DefaultValue="https://s3.ap-southeast-2.amazonaws.com/os.place.tech/outlook-plugin-resources/80x80-01.png"/>
       </bt:Images>
       <bt:Urls>
-        <bt:Url id="functionFile" DefaultValue="https://127.0.0.1:3000/function-file/function-file.html"/>
+      <bt:Url id="functionFile" DefaultValue="https://microsoft.poc.placeos.com/outlook/function-file/function-file.html"/> 
+        <bt:Url id="Taskpane.Url" DefaultValue="https://microsoft.poc.placeos.com/outlook/"/>
+        <bt:Url id="BookingsButton.Url" DefaultValue="https://microsoft.poc.placeos.com/outlook/upcoming"/>
+        <!-- <bt:Url id="functionFile" DefaultValue="https://127.0.0.1:3000/function-file/function-file.html"/>
         <bt:Url id="Taskpane.Url" DefaultValue="https://127.0.0.1:4200/"/>
-        <!-- <bt:Url id="Taskpane.Url" DefaultValue="https://portal-dev.placeos.run/#/"/> -->
+        <bt:Url id="BookingsButton.Url" DefaultValue="https://127.0.0.1:4200/upcoming"/> -->
```

Comment:
Should this be 127.0.0.1?

Reply:
> Should this be 127.0.0.1?

Oops, lines 100-102 should have been commented out, as they are outdated versions of lines 97-99 here. I'll remove these.